### ISSUE TITLE
feature request: autoplay

### ISSUE THREAD

**Issue by NoelDeMartin** [rosano/joybox#1]
I've wanted to listen to some of the lists you shared on [strolling](https://strolling.rosano.ca/), but it's cumbersome having to play songs one by one. It'd be nice if they play automatically instead.

**Comment by rosano** [comment]
Yes it has been suggested and I think it's a good idea that should be implemented.
